src/components/Timer/Timer.tsx

```diff
@@ -86,6 +91,18 @@ export const Timer = (props: TimerProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [timeLeft]);
 
+  useEffect(() => {
```

Comment:
What's going to happen if someone spams their ready button? 
Will the moderator receive limitless notifications about all users being ready?

Reply:
Yes this will happen.
I can look into it with a timeout and a state check, but imho its not relevant.